Incorporate feedback on OneLocBuild doc (#7522)
dotnet/arcade: Documentation/OneLocBuild.md

7. If using a mirrored repository (your code is mirrored to a trusted repository which your official build uses),
   add the following parameter to your YAML (subbing e.g. `sdk` for the value):
```yaml

>     MirrorRepo: {{name of your GitHub repository, e.g. 'sdk'}}

Reviewer: If this is the mirrored repo, maybe instead of saying it's the GitHub repository, it should be the name of the azure devops repository without the GitHub org prefix? that would work both for the case of `dotnet/sdk` which gets mirrored into `dotnet-sdk`, and `dotnet/msbuild`, which gets mirrored as `dotnet-msbuild-trusted`

If we're more specific in the example in this line, the following remark can probably be shortened a bit.

Author: It is the GitHub repository in all current cases, unfortunately. Deeply confusing, I know.

```
This naming might be confusing for repositories using code mirroring through Maestro, as typically the
"mirror repository" refers to the trusted Azure DevOps repository our GitHub repositories mirror to.

> In this case, however, it refers to the GitHub repository which is a _mirror of_ the Azure DevOps repository

Reviewer 2: ```suggestion
In this case, however, it refers to the repository that _mirrors to_ the Azure DevOps repository
```
Specifying github sounds like a good idea, I'm just not sure all repos that mirror would be GH repos. Maybe `it refers to the repository (likely GitHub) that mirrors to...`

Author: sounds good to me!
